### PR TITLE
Fix and simplify swisstopo source

### PR DESCRIPTION
**Pull request by gberaudo**
The projection was missing from the main typedef definition.

### CODE REVIEW

**Comment by gberaudo** [comment]
@sbrunner I reworked the PR and checked it is now displaying tiles correctly in EPSG:2056 projection.
Please have a new look if you want otherwise I can merge.

File: src/source/swisstoposource.js [hunk 1]
```diff
@@ -6,40 +6,72 @@ goog.require('ol.tilegrid.WMTS');
...
+ * @return {string} the url.
+ * @private
+ */
+ngeo.source.swisstopoCreateUrl_ = function(projection, format) {
+  if (projection === 'EPSG:2056') {
+    return 'https://wmts{10-14}.geo.admin.ch/1.0.0/{Layer}/default/{Time}' +
+      '/2056/{TileMatrix}/{TileCol}/{TileRow}.' + format;
+  } else if (projection === 'EPSG:21781') {
+    return 'https://wmts{5-9}.geo.admin.ch/1.0.0/{Layer}/default/{Time}' +
+      '21781/{TileMatrix}/{TileRow}/{TileCol}.' + format;
```
Comment (sbrunner) [inline src/source/swisstoposource.js]: Missing '/'!

File: src/source/swisstoposource.js [hunk 2]
```diff
@@ -53,24 +85,23 @@ ngeo.source.SwisstopoTileGrid21781_ = new ol.tilegrid.WMTS({
  * @export
  */
 ngeo.source.Swisstopo = function(options) {
-
   var format = options.format ? options.format : 'png';
-  var projection = (options.projection || 'EPSG:21781').toUpperCase();
-  var tilegrid = projection === 'EPSG:21781' ? ngeo.source.SwisstopoTileGrid21781_ :
-    ngeo.source.SwisstopoTileGrid2056_;
+  var projection = options.projection;
```
Comment (sbrunner) [inline src/source/swisstoposource.js]: Why not `options.projection || 'EPSG:21781'`?

File: src/source/swisstoposource.js [hunk 3]
```diff
@@ -6,40 +6,72 @@ goog.require('ol.tilegrid.WMTS');
 
 
 /**
+ * Available resolutions as defined in
+ * http://api3.geo.admin.ch/services/sdiservices.html#wmts .
```
Comment (sbrunner) [inline src/source/swisstoposource.js]: Extra space before dot?

File: src/source/swisstoposource.js [hunk 4]
```diff
@@ -6,40 +6,72 @@ goog.require('ol.tilegrid.WMTS');
...
-  1000, 750, 650, 500, 250, 100, 50, 20, 10, 5, 2.5, 2, 1.5, 1, 0.5
+  1000, 750, 650, 500, 250, 100, 50, 20, 10, 5, 2.5, 2, 1.5, 1, 0.5,
+  0.25, 0.1
 ];
 
 
 /**
- * @const {ol.tilegrid.WMTS}
+ * The matrix set is is constructed by passing the matrix set defined in the
+ * table at http://api3.geo.admin.ch/services/sdiservices.html#wmts .
```
Comment (sbrunner) [inline src/source/swisstoposource.js]: Same...

File: src/source/swisstoposource.js (hunk 1 above)
Comment (gberaudo) [inline src/source/swisstoposource.js]: Thanks

**Comment by gberaudo** [comment]
Thanks for the review.

**Pull request merged**